sdk/src/main/scala/com/horizen/account/api/rpc/service/EthService.scala

```diff
@@ -786,6 +794,32 @@ class EthService(
         // apply requested transaction with tracing enabled
         tagStateView.applyTransaction(requestedTx, previousTransactions.length, gasPool, blockContext)
 
+        // return the tracer result from the evm
+        if(blockContext.getEvmResult != null) {
+          blockContext.getEvmResult.tracerResult
+        } else Unit
+      }
+    }
+  }
+
+  @RpcMethod("debug_traceCall")
+  @RpcOptionalParameters(1)
+  def traceCall(params: TransactionArgs, tag: String, config: TraceOptions): Any = {
+
+    applyOnAccountView { nodeView =>
+      // get block to trace
+      val (block, blockInfo) = getBlockById(nodeView, getBlockIdByHashOrTag(nodeView, tag))
```

Comment:
One last note:
`block` is unused now and `blockInfo` is only used to get the block height, which is just used to lookup the block hash again :sweat_smile: 
There is potential to refactor here, but could also do that later.

Reply:
Did a small refactor adding and calling a method that retrieve only the `blockInfo` by _id_ and not the entire `block`